src/functions/severa/get-flextime-by-user/handler.ts

```diff
@@ -0,0 +1,38 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import type FlextimeModel from "src/database/models/severa";
+import { CreateSeveraApiService } from "src/database/services/severa-api-service";
+
+/**
+ * @param event
+ * @returns users flextime
+ */
+export const getFlextimeHandler: APIGatewayProxyHandler = async (
+  event: APIGatewayProxyEvent,
+) => {
+  try {
+    const userGuid = event.pathParameters?.userGuid;
+    const eventDate = event.queryStringParameters?.eventDate;
```

Comment:
As commented in the spec PR we should set the date to be the current date for now, we just want to know todays/ this months balance.

Reply:
Meaning eventDate can be removed- we should check what the default value is set as by the severa API as it is not a required param in their docs